### PR TITLE
Victor Kalinkin css-pseudo-classes (initial submission)

### PR DESCRIPTION
**Pull request by vkalinkin**
css pseudo class

### CODE REVIEW

File: css-pseudo-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+table {
+  border: 2px solid white;
+}
+
+th {
+  background-color: black;
+  color: white;
+}
+
+.even {
```
Comment (Zhiwutian) [inline css-pseudo-classes/styles.css:20]: you want to use the nth-child pseudo class to complete this. Try refactoring using that!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed!

**Comment by vkalinkin** [comment]
css pseudo class nth child selector

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**